entries/a/ally.com.json

```
@@ -3,8 +3,7 @@
     "domain": "ally.com",
     "url": "https://www.ally.com/",
     "tfa": [
-      "sms",
-      "email"
+      "sms"
```

According to #7033 automated phone calls are also being used.
```suggestion
      "sms",
      "call"
```